src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +76,17 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<ReadOnlyPerson> sortedListView() {
+        this.internalList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                String name1=o1.getName().toString();
+                String name2=o2.getName().toString();
```

Comment:
For this line of code and the line before, writing "String name2 = o2.getName().toString();" with the spaces rather than doing what is being done now.